internal/v1/server_test.go

```diff
@@ -26,7 +26,23 @@ import (
 	"github.com/labstack/echo/v4"
 )
 
-var UUIDTest = uuid.MustParse("d1f631ff-b3a6-4eec-aa99-9e81d99bc93d")
+var dbc *tutils.PSQLContainer
+
+func TestMain(m *testing.M) {
+	d, err := tutils.NewPSQLContainer()
+	if err != nil {
+		panic(err)
+	}
+
+	dbc = d
+	code := m.Run()
+
+	err = dbc.Stop()
```

Comment:
I wonder whether we can somehow put this into a `defer` block. I would love making this as error-proof as possible, as I don't like the sound of dangling database containers on my system.